src/main/java/com/everis/cars/timer/ActivateCarTimer.java

```diff
@@ -0,0 +1,76 @@
+package com.everis.cars.timer;
+
+import java.util.List;
+
+import javax.ejb.EJB;
+import javax.ejb.Schedule;
+import javax.ejb.Singleton;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.TypedQuery;
+
+import org.apache.log4j.Logger;
+
+import com.everis.cars.control.CarService;
+import com.everis.cars.entity.Car;
+
+/**
+ * This class check for {@link Car}s with false status 
+ * and change their status to true
+ */
+@Singleton
+public class ActivateCarTimer {
+	
+	/**
+	 * {@link Logger} instance
+	 * 
+	 * @see org.apache.log4j.Logger
+	 */
+	private static Logger logger = Logger.getLogger(ActivateCarTimer.class);
+	
+	/**
+	 * Car Service injected
+	 * 
+	 * @see com.everis.cars.control.CarService
+	 */
+	@EJB
+	CarService carService;
+
+	/**
+	 * {@link EntityManager} dependency injected to the service
+	 * 
+	 * @see javax.persistence.EntityManager
+	 */
+	@PersistenceContext(unitName = "em_postgres")
```

Comment:
this should not be here, only PersistenceService should inject persistenceContext

Reply:
Removed PersistenceService from ActivateCarTimer.